distribution/src/resources/distrib-auto.sh

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
```

Comment:
In general, "distrib" files are meant to be added by Drill distributions, not by Drill itself. If we want to provide the memory checking in Drill, then it should go into some other file other than "distrib."

Reply:
Wasn't sure which would be a a suitable place. I reasoned the "Apache" distribution is what we are defining it for, hence the choice